tests/integration/issues/github_976/test_topk_rest_api.py

```diff
@@ -1,22 +1,23 @@
-import pytest
 import json
+import pytest
+import random
 from urllib import request
 
 from jina.flow import Flow
 
 
 TOP_K = 2
-
+PORT = random.randint(1, 10000)
```

Comment:
we have a random port method built-in codebase 
```
from jina.helper import random_port
```

Reply:
yes, the reason of not randomint is many ports below 40000 are reserved by OS already. Example: 22, 80